inspirehep/config.py

```diff
@@ -827,8 +828,10 @@ def _(x):
                              ':json_serializer'),
     },
     search_serializers={
-        'application/json': ('invenio_workflows_ui.serializers'
+        'application/vnd+inspire.brief+json': ('invenio_workflows_ui.serializers'
```

Comment:
Eventually I guess we will directly use the invenio-search angular module instead of the inspire-search and this is not needed?